drivers/backup/auth.go

```diff
@@ -177,6 +178,13 @@ type KeycloakGroupToUser struct {
 	Realm   string `json:"realm"`
 }
 
+// AddRoleToKeyCloak representation of adding role to keycloak
+type AddRoleToKeyCloak struct {
```

Comment:
Can we call this just `Role` ?

Reply:
we already have role struct backup.go , hence changes this into RoleCreate.